tools/c7n_mailer/README.md

```diff
@@ -248,7 +248,8 @@ The following extra jinja filters are available:
 
 | filter | behavior |
 |:----------|:-----------|
-| `{{ utc_string|date_time_format(tz_str='US/Pacific', format='%Y %b %d %H:%M %Z') }}` | pretty [format](https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior) the date / time |
+| <code>utc_string&#124;date_time_format(tz_str='US/Pacific', format='%Y %b %d %H:%M %Z')</code> | pretty [format](https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior) the date / time |
```

Comment:
This fixes a rendering issue with markdown on the README. I didn't realize I couldn't use a real pipe, since that messed up how it's viewed. I switched to using the html code for a pipe: &#124;